nucleus/__init__.py

```diff
@@ -1101,9 +1102,14 @@ def delete_model(self, model_id: str) -> dict:
         )
         return response
 
-    def create_custom_index(self, dataset_id: str, embeddings_url: str):
+    def create_custom_index(
```

Comment:
Consider adding a doc string here. Methods defined here are part of the public API.